### PR TITLE
[DFFE-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by lorlahdey**
@Ireti203

### CODE REVIEW

**Comment by lorlahdey** [comment]
@Ireti203

File: submissions/lawalLolade/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	product_description VARCHAR(255) NOT NULL,
+	product_image BLOB(255) NOT NULL,
+	unit_price DECIMAL (15,2) NOT NULL,
+	stock_level VARCHAR(20) NOT NULL,
+	product_status VARCHAR(50) NOT NULL,
+	created_at DATETIME NOT NULL,
+	PRIMARY KEY (id),
+	FOREIGN KEY (admin_users_id) REFERENCES admin_users (id),
+	FOREIGN KEY (product_categories_id) REFERENCES product_categories (id)
+);
```
Comment (adaugochi) [inline submissions/lawalLolade/database/init.sql:55]: Insert some data into the products table

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by lorlahdey** [comment]
@adaugochi

File: submissions/lawalLolade/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+
+INSERT INTO products (admin_users_id, product_categories_id, product_name, product_description, product_image, unit_price, stock_level, product_status, created_at)
+VALUES
+	(1, 2, 'Cherry Blossom', 'A feminine arrangement of pale pink Asiatic Lilies alongside hot pink carnations, pink and white two tone spray carnations and white Freesias.', 'C:\Users\Mr L.O. Lawal\Desktop\Dufuna-Fem\submissions\lawalLolade\HTML\images\Prod2.jpeg', 15000, 8, 'in stock', now()),
```
Comment (adaugochi) [inline submissions/lawalLolade/database/init.sql:60]: use image URL rather than pointing to a path on your local machine

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by lorlahdey** [comment]
@adaugochi @Ireti203

**Comment by adaugochi** [comment]
> 
> 
> @adaugochi @Ireti203

use image URL rather than pointing to a path on your local machine

**Comment by lorlahdey** [comment]
I made the changes already

**Review by adaugochi — CHANGES_REQUESTED** [review]
Instead of doing this `'C:\Users\Mr L.O. Lawal\Desktop\Dufuna-Fem\submissions\lawalLolade\HTML\images\Prod2.jpeg` use an image url from google

**Comment by lorlahdey** [comment]
@adaugochi

**Review by adaugochi — APPROVED** [review]

**Pull request merged**